internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,64 @@
+// Пакет daysteps формирует сводку после дневной прогулки
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
+// Parse заполняет данные о прошедшей тренировке.
+// При недействительных значениях выводит ошибку.
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	data := strings.Split(datastring, ",")
+	if len(data) != 2 {
+		return errors.New("data is invalid")
+	}
+
+	ds.Steps, err = strconv.Atoi(data[0])
+	if err != nil {
+		log.Println(err)
+		return err
```

Comment:
Лучше всегда оборачивать ошибку
```suggestion
		return fmt.Errorf("invalid steps: %w", err)
```